cluster/gke/config-common.sh

```diff
@@ -21,6 +21,7 @@
 MASTER_NAME="k8s-${CLUSTER_NAME}-master"
 ZONE="${ZONE:-us-central1-f}"
 NUM_MINIONS="${NUM_MINIONS:-2}"
+MINION_NAMES=($(eval echo "k8s-${CLUSTER_NAME}-node-{1..${NUM_MINIONS}}"))
```

Comment:
It's using the range expansion voodoo in bash (
http://wiki.bash-hackers.org/syntax/expansion/brace#ranges ), similar to
how cluster/gce/config-default.sh is doing it.

$ echo foo-{1..3}
foo-1 foo-2 foo-3

On Wed, Jan 14, 2015 at 12:38 PM, Daniel Smith <notifications@github.com>
wrote:

> In cluster/gke/config-common.sh
> <https://github.com/GoogleCloudPlatform/kubernetes/pull/3473#discussion-diff-22967972>
> :
>
> > @@ -21,6 +21,7 @@
> >  MASTER_NAME="k8s-${CLUSTER_NAME}-master"
> >  ZONE="${ZONE:-us-central1-f}"
> >  NUM_MINIONS="${NUM_MINIONS:-2}"
> > +MINION_NAMES=($(eval echo "k8s-${CLUSTER_NAME}-node-{1..${NUM_MINIONS}}"))
>
> I'm no bash expert, but should {1..${NUM_MINIONS}} have a $ in front?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/GoogleCloudPlatform/kubernetes/pull/3473/files#r22967972>
> .
>